models/core/transfers.sql

```diff
@@ -0,0 +1,47 @@
+{{ 
+    config(
+        materialized='incremental', 
+        unique_key= 'log_id',
+        incremental_strategy = 'delete+insert',
+        tags=['core'],
+        cluster_by = ['block_timestamp']
+    ) 
+}}
+
+with logs as (
```

Comment:
```suggestion
with

logs as (
```

Reply:
This was suggested by @chuxinh. So which one should I follow?

Reply:
Well... Firstly, sorry for all these nit picking...

Personally my style is like this... 
https://github.com/antonyip/harmony_dbt/blob/experiments/models/playground/ant_view_token_price_gaia.sql
which is totally against their style guide.. but works for me as its easy to comment rows out..

Back to the style guide.. it shows "WITH" on its own row..
https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md

I think chuxin might have made a small mistake here.. 

Otherwise.. Good job on this PR! I've no other suggestions to make :)